doc/api/process.md

```diff
@@ -1651,7 +1651,9 @@ added: v0.2.0
 * {Object}
 
 The `process.versions` property returns an object listing the version strings of
-Node.js and its dependencies.
+Node.js and its dependencies. In addition, `process.versions.modules` indicates
+the current ABI version, which is increased whenever a C++ API changes. Node.js
+will refuse to load native modules built for an older `modules` value.
```

Comment:
Right, so text should be something like "Node.js will refuse to load modules that weren't compiled against its own
 module ABI number."